src/Compilers/CSharp/Portable/FlowAnalysis/PreciseAbstractFlowPass.cs

```diff
@@ -1293,6 +1293,11 @@ public override BoundNode VisitLiteral(BoundLiteral node)
             return null;
         }
 
+        public override BoundNode VisitMethodToken(BoundMethodToken node)
+        {
+            return null;
```

Comment:
This probably means that instrumentation is done too early. It should be done after flow analysis.